### PR TITLE
Show date range in atlas

### PR DESCRIPTION
**Pull request by laurence-hudson-tessella**

### CODE REVIEW

**Comment by edwiles** [comment]
Update DB diagram?

File: db/migrations/1501261621__add_occurrence_date_range_to_model_run.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+-- Updates the database schema to add occurrence_data_range_start_date and occurrence_data_range_end_date columns to the model_run table.
+-- Copyright (c) 2015 University of Oxford
+
+ALTER TABLE model_run ADD COLUMN occurrence_data_range_start_date timestamp;
+ALTER TABLE model_run ADD COLUMN occurrence_data_range_end_date timestamp;
+
+-- Set all old model runs to "now", as old model runs will be deleted and the data range is not retrivable for non-automatic historic runs.
```
Comment (edwiles) [inline db/migrations/1501261621__add_occurrence_date_range_to_model_run.sql]: retrievable

File: src/Common/src/uk/ac/ox/zoo/seeg/abraid/mp/common/domain/ModelRun.java [hunk 2]
```diff
@@ -143,6 +143,17 @@
...
+    @Column(name = "occurrence_data_range_start_date")
+    @Type(type = "org.jadira.usertype.dateandtime.joda.PersistentDateTime")
+    private DateTime occurrenceDataRangeStartDate;
+
+    // The occurrence date of the latest associated disease occurrence.
+    @Column(name = "occurrence_data_range_end_date")
+    @Type(type = "org.jadira.usertype.dateandtime.joda.PersistentDateTime")
+    private DateTime occurrenceDataRangeEndDate;
+
+
```
Comment (edwiles) [inline src/Common/src/uk/ac/ox/zoo/seeg/abraid/mp/common/domain/ModelRun.java]: 2 blank lines

File: src/PublicSite/web/js/app/atlas/LegendViewModel.js [hunk 3]
```diff
@@ -9,5 +9,13 @@ define(["ko"], function (ko) {
     return function () {
         var self = this;
         self.type = ko.observable().subscribeTo("active-atlas-type");
+
+        var activeRun = ko.observable().subscribeTo("selected-run");
+        self.startDate =  ko.computed(function () {
```
Comment (edwiles) [inline src/PublicSite/web/js/app/atlas/LegendViewModel.js]: Double space (also line 17)

**Comment by edwiles** [comment]
LGTM

File: src/Common/test/uk/ac/ox/zoo/seeg/abraid/mp/common/dao/ModelRunDaoTest.java [hunk 4]
```diff
@@ -68,9 +68,12 @@ public void saveAndReloadModelRunByName() {
         DateTime batchStartDate = DateTime.now().minusHours(6).minusDays(1);
         DateTime batchEndDate = DateTime.now().minusHours(6);
         DateTime batchingCompletionDate = DateTime.now().plusHours(2);
+        DateTime occurrenceDataRangeStartDate = DateTime.now().plusHours(9);
+        DateTime occurrenceDataRangeEndDate = DateTime.now().plusHours(8);
```
Comment (helena-patching-tessella) [inline src/Common/test/uk/ac/ox/zoo/seeg/abraid/mp/common/dao/ModelRunDaoTest.java]: It isn't so critical in this test situation, but here start date > end date. Could this (or should this) have some validation or constraint on it somewhere?

**Comment by helena-patching-tessella** [comment]
LGTM

**Pull request merged**